FLIR/conservator/cli/managers.py

```diff
@@ -427,7 +427,8 @@ def upload(localpath, remote_collection, remote_name, create_collections):
                 collection = conservator.collections.from_string(
                     remote_collection, fields="id"
                 )
-            conservator.upload(
+            media_manager = MediaTypeManager(conservator)
```

Comment:
`conservator.media.upload` would work here instead of creating a new manager instance